go/vt/mysqlctl/azblobbackupstorage/azblob.go

```diff
@@ -276,15 +276,20 @@ func (bs *AZBlobBackupStorage) containerURL() (*azblob.ContainerURL, error) {
 
 // ListBackups implements BackupStorage.
 func (bs *AZBlobBackupStorage) ListBackups(ctx context.Context, dir string) ([]backupstorage.BackupHandle, error) {
-	log.Infof("ListBackups: [azblob] container: %s, directory: %v", *containerName, objName(dir, ""))
+	var searchPrefix string
+	if dir == "/" {
+		searchPrefix = "/"
+	} else {
+		searchPrefix = objName(dir, "")
+	}
+
+	log.Infof("ListBackups: [azblob] container: %s, directory: %v", *containerName, searchPrefix)
```

Comment:
Is this log line required? will it spam the logs?

Reply:
this log was originally in there, I just moved it below the searchPrefix assignment. I'm fine either way to remove it or not

Reply:
we can leave it in and remove it if it become noisy.